src/coreclr/vm/pendingload.h

```diff
@@ -166,7 +166,7 @@ class PendingTypeLoadEntry
     }
 #endif //DACCESS_COMPILE
 
-    TypeKey GetTypeKey()
+    TypeKey& GetTypeKey()
```

Comment:
GCC wanted & here, since we pass the result by reference. Not sure how it worked with MSVC.